src/main/java/org/fcrepo/importexport/exporter/Exporter.java

```diff
@@ -165,6 +168,10 @@ public Exporter(final Config config, final FcrepoClient.FcrepoClientBuilder clie
 
                 //enforce default metadata
                 bagProfile.validateConfig(bagConfig);
+                final Map<String, String> profileMetadata = bagProfile.getProfileMetadata();
+
+                // assume this is already valid?
```

Comment:
Given that the bag profiles are included in the codebase, it would be ideal to have a test which validated all the profiles on build so that we can be confident they are valid and this assumption wouldn't need to be a question. Adding such a test isn't required for this PR, but I would suggest changing this comment to a statement

Reply:
Updated the wording in 71ac512. [FCREPO-3204](https://jira.lyrasis.org/browse/FCREPO-3204) will end up taking care of the tests to validate the bag profiles included in the utility.